src/includes/capture-error/native.mdx

```diff
@@ -1,23 +1,30 @@
 To capture an error or exception condition, create events containing an
-exception object. It needs to contain at least a value and type:
+exception object:
 
 ```cpp
 #include <sentry.h>
 
-sentry_value_t exc = sentry_value_new_object();
-sentry_value_set_by_key(exc, "type", sentry_value_new_string("Exception"));
-sentry_value_set_by_key(exc, "value", sentry_value_new_string("Error message."));
+sentry_value_t event = sentry_value_new_event();
+
+sentry_value_t exc = sentry_value_new_exception("Exception", "Error message.");
+sentry_event_add_exception(event, exc);
 
-sentry_value_t exceptions = sentry_value_new_object();
-sentry_value_t values = sentry_value_new_list();
+sentry_capture_event(event);
+```
 
-sentry_value_set_by_key(exceptions, "values", values);
-sentry_value_append(values, exc);
+The above exception does not contain a stack trace, which must be added separately:
+
+```cpp
+#include <sentry.h>
 
 sentry_value_t event = sentry_value_new_event();
-sentry_value_set_by_key(event, "exception", exceptions);
+
+sentry_value_t exc = sentry_value_new_exception("Exception", "Error message.");
+
+sentry_value_t stacktrace = sentry_value_new_stacktrace(NULL, 0);
+sentry_value_set_by_key(exc, "stacktrace", stacktrace);
```

Comment:
I think it was these two which we could replace with another function:

```void sentry_value_add_stacktrace(sentry_value_t, void**, size_t)```

to get rid of key/entry management on the client-side, correct?